static/styles/jsdoc-default.css

```diff
@@ -1,569 +1,606 @@
 * {
```

Comment:
This file seems to have only formatting changes, can you please revert it to make it easier to review?  (or if there are changes other than formatting, they should be visible)

Reply:
I added the format to change for class `returns` which I defined in `returns.tmpl`. Others are formatting changes, I reverted now.

Reply:
Thank you, the changes are much more visible now.